### PR TITLE
Simple update for BAR

### PR DESCRIPTION
**Pull request by qknight**
So we make sure developers don't get confused with the fork base and our repo

### CODE REVIEW

**Comment by Beherith** [comment]
Thanks! +1

**Comment by badosu** [comment]
Consider pointing to `BAR105` instead of `master`

File: README.markdown [hunk 1]
```diff
@@ -1,42 +1,45 @@
-# Spring RTS game engine
-[![Build Status](https://travis-ci.org/spring/spring.svg?branch=develop)](https://travis-ci.org/spring/spring)
+# BeyondAllReason RTS game engine
 
-## README
+    git clone https://github.com/beyond-all-reason/spring
```
Comment (lhog) [inline README.markdown:3]: `git clone https://github.com/beyond-all-reason/spring -b BAR105`

File: README.markdown [hunk 2]
```diff
@@ -1,42 +1,45 @@
...
 
-Spring (formerly TASpring) is an Open Source Real Time Strategy game engine.
-Visit our [project homepage](https://springrts.com/) for help, suggestions,
+BeyondAllReason a fork of [Open Source Real Time Strategy game engine](https://github.com/spring/spring) (formerly Spring/TASpring).
```
Comment (lhog) [inline README.markdown:5]: BeyondAllReason **is** a fork.

File: README.markdown (hunk 1 above)
Comment (qknight) [reply README.markdown:3]: But isn't that outdated in like no time?

**Comment by qknight** [comment]
Incorperated both requested changes. Once this has been merged i'll create another PR for the development branch BAR105

**Comment by badosu** [comment]
Merging to master will be moot or an issue, as either it will generate a conflict when rebasing whatever we migrate to master from or will be overriden

**Comment by qknight** [comment]
> Merging to master will be moot or an issue, as either it will generate a conflict when rebasing whatever we migrate to master from or will be overriden

That is true. But maybe then we can just trash master and recreate the branch from the most recent 'branch', which is BAR105 at time of writing.

**Comment by lhog** [comment]
Note for the future, you should open the PR against the default branch, in this case BAR105.
I manually ported your commit there and delete from `master` which is currently unused

**Pull request merged**